### DISCUSSION TITLE
Flashforge Guider iis

### DISCUSSION THREAD

**Discussion by mroland72** [OrcaSlicer/OrcaSlicer#2337] [Q&A]
Does anyone happen to have a profile for the Flashforge Guider iis for OrcaSlicer?  I see the one for the Adventurer 5 but nothing else.

**Comment by RafBois** [comment]
Yes, it would be nice if FlashForge would open up a bit as their FlashPrint slicer is really very limited!!!

I'm also looking for a profile for a FlashForge Creator Pro in Klipper.

**Reply by mroland72** [reply]
Not looking promissing.

**Reply by Dcnigma** [reply]
Guider II too please :-)

**Reply by AC5SHOT** [reply]
could not agree more

**Reply by AllanFab** [reply]
Flashforge Creator 3 Please
